cl/search/models.py

```diff
@@ -423,6 +439,26 @@ class Docket(AbstractDateTimeModel, DocketSources):
         null=True,
         related_name="child_dockets",
     )
+    transfer_date = models.DateField(
+        help_text="The date the case was transfered to another court if applicable",
+        db_index=True,
```

Comment:
This can be revisited, but dates have been indexed as they seem likely to be a fairly common part of queries.